src/System.Private.DataContractSerialization/src/System/Runtime/Serialization/DataContract.cs

```diff
@@ -2106,10 +2106,7 @@ private static void ImportKnownTypeAttributes(Type type, Dictionary<Type, Type>
                         {
                             knownDataContracts = new DataContractDictionary();
                         }
-                        if (!knownDataContracts.ContainsKey(itemDataContract.StableName))
-                        {
-                            knownDataContracts.Add(itemDataContract.StableName, itemDataContract);
-                        }
+                        knownDataContracts.TryAdd(itemDataContract.StableName, itemDataContract);
```

Comment:
I think we generally try not to have a blank line after a close brace except when it's another close brace, or an `else`. Although I don't see that in https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md